src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -297,6 +297,40 @@ public void CloseSignalingCommandsMenu()
             stage.HUD.ApplySignalCommand(command, stage.Player);
     }
 
+    [RunOnKeyDown("g_sprint")]
+    public bool StartSprint()
+    {
+        if (!stage.HasPlayer || !stage.GameWorld.WorldSettings.ExperimentalFeatures)
+            return true;
```

Comment:
Shouldn't this *not* consume input, this doesn't do anything except cause other things to not be able to react to this keypress?